package.json

```diff
@@ -13,7 +13,7 @@
     "relay2ts": "relay2ts src/components/*.tsx --update",
     "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && yarn install && npm run dump-schema -- ../../data",
     "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
-    "lint": "tslint -c tslint.json --project tsconfig.json",
+    "lint": "tslint -c tslint.json --project tsconfig.json --fix",
```

Comment:
Right that's a good point... let's remove the auto-fixing and just run it ad-hoc then.